verl/workers/config/engine.py

```diff
@@ -113,8 +123,17 @@ class FSDPEngineConfig(BaseConfig):
     entropy_from_logits_with_chunking: bool = False
     use_torch_compile: bool = True
     entropy_checkpointing: bool = False
-    forward_only: bool = False
-    strategy: str = "fsdp"
+    strategy: str = field(default="fsdp", init=False)
 
     def __post_init__(self):
         assert self.strategy in ["fsdp", "fsdp2"], f"strategy {self.strategy} not supported"
+
+
+@dataclass
+class AllEngineConfig(FSDPEngineConfig, McoreEngineConfig):
```

Comment:
This is undesired. We will start to expand the engine to veomni, torchtitan, etc...